src/kv/committable_tx.h

```diff
@@ -193,6 +190,12 @@ namespace kv
           }
         }
 
+        if (version == NoVersion)
```

Comment:
This is to return even earlier in case there's nothing to serialise, i.e. read-only Tx.